diff --git a/node_modules/ssb-atala-prism/graphql/resolvers.js b/node_modules/ssb-atala-prism/graphql/resolvers.js
index 06f552f..590ce0c 100644
--- a/node_modules/ssb-atala-prism/graphql/resolvers.js
+++ b/node_modules/ssb-atala-prism/graphql/resolvers.js
@@ -31,6 +31,7 @@ module.exports = function (ssb) {
       },
 
       sendCredentialProof (parent, { invitationUrl, credentialId }) {
+        console.log('==== sendCredentialProof hit =====')
         return ssb.atalaPrism.sendCredentialProof(invitationUrl, credentialId)
           .catch(err => { throw new GraphQLError(err) })
       }
diff --git a/node_modules/ssb-atala-prism/index.js b/node_modules/ssb-atala-prism/index.js
index 09aafd5..d81feed 100644
--- a/node_modules/ssb-atala-prism/index.js
+++ b/node_modules/ssb-atala-prism/index.js
@@ -362,7 +362,12 @@ module.exports = {
       // => verifier accepts connection, then issues presentation request
       // (webhook: verifier pushes update to server, which can then progress
       // to sending presentation request)
-      acceptConnectionInvitation(invitationUrl)
+      // accept connectionInvitation request
+      await acceptConnectionInvitation(invitationUrl)
+        .catch(err => {
+          console.log(err)
+          cb(err)
+        })
     }
 
     return {
