diff --git a/node_modules/artefact-server/index.js b/node_modules/artefact-server/index.js
index 56a65b1..23275ad 100644
--- a/node_modules/artefact-server/index.js
+++ b/node_modules/artefact-server/index.js
@@ -179,11 +179,11 @@ async function readBlobRequest (ctx) {
 
 function AddFileToStore (store, host, port) {
   return async function addFileToStore (fileStream, fileName, fileEncryptionKey) {
-    if (!fileEncryptionKey) throw new Error(`addFileToStore requires a fileEncryptionKey, got ${fileEncryptionKey}`)
-
-    const result = await store.addFileStream(fileStream, fileName, {
-      optionalFileEncryptionKey: Buffer.from(fileEncryptionKey, 'hex')
-    })
+    const opts = {}
+    if (fileEncryptionKey) {
+      opts.optionalFileEncryptionKey = Buffer.from(fileEncryptionKey, 'hex')
+    }
+    const result = await store.addFileStream(fileStream, fileName, opts)
 
     const url = new URL(`http://${host}:${port}/drive/${result.driveAddress}/blob/${result.fileName}`)
     url.search = new URLSearchParams({
