diff --git a/node_modules/ssb-ahau/src/graphql/index.js b/node_modules/ssb-ahau/src/graphql/index.js
index b4750e7..d37662a 100644
--- a/node_modules/ssb-ahau/src/graphql/index.js
+++ b/node_modules/ssb-ahau/src/graphql/index.js
@@ -84,12 +84,12 @@ module.exports = async function graphqlServer (ssb) {
     cordova-bridge is used in Cordova, DO NOT REMOVE!
     The if statement is uncommented by patch-package in the mobile platform
   */
-  // if (process.env.PLATFORM === 'cordova') {
-  //   require('cordova-bridge').channel.post(
-  //     'initialized',
-  //     JSON.stringify({ started: true })
-  //   )
-  // }
+  if (process.env.PLATFORM === 'cordova') {
+    require('cordova-bridge').channel.post(
+      'initialized',
+      JSON.stringify({ started: true })
+    )
+  }
 
   return httpServer
 }
diff --git a/node_modules/ssb-ahau/src/graphql/stats.js b/node_modules/ssb-ahau/src/graphql/stats.js
index fe98fa0..d0e1e3b 100644
--- a/node_modules/ssb-ahau/src/graphql/stats.js
+++ b/node_modules/ssb-ahau/src/graphql/stats.js
@@ -1,5 +1,4 @@
 const gql = require('graphql-tag')
-const si = require('systeminformation')
 const fs = require('fs')
 const path = require('path')
 
@@ -78,13 +77,9 @@ function Resolvers (ssb, externalGetters) {
   const resolvers = {
     Query: {
       fileSystemStats: async () => {
-        const fsSize = await si.fsSize()
-        const sizes = fsSize
-          .filter(fs => appPath.startsWith(fs.mount))
-          .sort((a, b) => b.mount.length - a.mount.length) // longest first
-
-        if (!sizes.length) return null
-        return sizes[0]
+        const err = Error('fileSystemStats not yet supported in mobile')
+        console.log(err)
+        throw err
       },
       ahauDBStats: async () => {
         return {}
