diff --git a/node_modules/ssb-ahau/src/graphql/index.js b/node_modules/ssb-ahau/src/graphql/index.js
index c1341af..690f753 100644
--- a/node_modules/ssb-ahau/src/graphql/index.js
+++ b/node_modules/ssb-ahau/src/graphql/index.js
@@ -87,12 +87,12 @@ module.exports = async function graphqlServer (ssb) {
     cordova-bridge is used in Cordova, DO NOT REMOVE!
     The if statement is uncommented by patch-package in the mobile platform
   */
-  // if (process.env.PLATFORM === 'cordova') {
-  //   require('cordova-bridge').channel.post(
-  //     'initialized',
-  //     JSON.stringify({ started: true })
-  //   )
-  // }
+  if (process.env.PLATFORM === 'cordova') {
+    require('cordova-bridge').channel.post(
+      'initialized',
+      JSON.stringify({ started: true })
+    )
+  }
 
   return httpServer
 }
diff --git a/node_modules/ssb-ahau/src/graphql/stats.js b/node_modules/ssb-ahau/src/graphql/stats.js
index fe98fa0..a60d05a 100644
--- a/node_modules/ssb-ahau/src/graphql/stats.js
+++ b/node_modules/ssb-ahau/src/graphql/stats.js
@@ -1,5 +1,4 @@
 const gql = require('graphql-tag')
-const si = require('systeminformation')
 const fs = require('fs')
 const path = require('path')
 
@@ -77,15 +76,7 @@ function Resolvers (ssb, externalGetters) {
 
   const resolvers = {
     Query: {
-      fileSystemStats: async () => {
-        const fsSize = await si.fsSize()
-        const sizes = fsSize
-          .filter(fs => appPath.startsWith(fs.mount))
-          .sort((a, b) => b.mount.length - a.mount.length) // longest first
-
-        if (!sizes.length) return null
-        return sizes[0]
-      },
+      fileSystemStats: () => null,
       ahauDBStats: async () => {
         return {}
       },
