diff --git a/node_modules/ssb-ahau/index.js b/node_modules/ssb-ahau/index.js
index c3aebf1..bd898be 100644
--- a/node_modules/ssb-ahau/index.js
+++ b/node_modules/ssb-ahau/index.js
@@ -2,7 +2,7 @@ const pull = require('pull-stream')
 const Obz = require('obz')
 const graphqlServer = require('./src/graphql')
 const autoPrivateProfile = require('./src/auto-private-profile')
-const autoPrune = require('./src/auto-prune')
+// const autoPrune = require('./src/auto-prune')
 
 module.exports = {
   name: 'ahau',
@@ -20,7 +20,7 @@ module.exports = {
 
     autoPrivateProfile(sbot, isReady)
 
-    autoPrune(sbot, isReady)
+    // autoPrune(sbot, isReady)
 
     graphqlServer(sbot, (err, graphql) => {
       if (err) throw err
diff --git a/node_modules/ssb-ahau/src/graphql/index.js b/node_modules/ssb-ahau/src/graphql/index.js
index d71f69e..35484b7 100644
--- a/node_modules/ssb-ahau/src/graphql/index.js
+++ b/node_modules/ssb-ahau/src/graphql/index.js
@@ -20,7 +20,7 @@ const Settings = require('@ssb-graphql/settings')
 const Author = require('./author')
 const Backup = require('./backup')
 const LatestSequence = require('./latest-sequence')
-const StorageManagement = require('./storage-management')
+// const StorageManagement = require('./storage-management')
 
 module.exports = function graphqlServer (sbot, cb) {
   const PORT = (sbot.config.graphql && sbot.config.graphql.port) || env.graphql.port
@@ -42,7 +42,7 @@ module.exports = function graphqlServer (sbot, cb) {
   const invite = Invite(sbot)
   const backup = Backup(sbot)
   const latestSequence = LatestSequence(sbot)
-  const storage = StorageManagement(sbot)
+  // const storage = StorageManagement(sbot)
 
   if (sbot.config.graphql && sbot.config.graphql.loadContext === false) {
     return next(null)
@@ -63,7 +63,7 @@ module.exports = function graphqlServer (sbot, cb) {
       settings,
       author,
       invite,
-      storage,
+      // storage,
       latestSequence,
       backup
     ])
@@ -123,12 +123,12 @@ module.exports = function graphqlServer (sbot, cb) {
         cordova-bridge is used in Cordova, DO NOT REMOVE!
         The if statement is uncommented by patch-package in the mobile platform
       */
-      // if (process.env.PLATFORM === 'cordova') {
-      //   require('cordova-bridge').channel.post(
-      //     'initialized',
-      //     JSON.stringify({ started: true })
-      //   )
-      // }
+      if (process.env.PLATFORM === 'cordova') {
+        require('cordova-bridge').channel.post(
+          'initialized',
+          JSON.stringify({ started: true })
+        )
+      }
 
       cb(null, httpServer)
     })
