diff --git a/node_modules/ssb-ahau/src/graphql/index.js b/node_modules/ssb-ahau/src/graphql/index.js
index 2371f08..6c56a53 100644
--- a/node_modules/ssb-ahau/src/graphql/index.js
+++ b/node_modules/ssb-ahau/src/graphql/index.js
@@ -21,7 +21,6 @@ const Settings = require('@ssb-graphql/settings')
 const Author = require('./author')
 const GroupInit = require('./group-init')
 const Backup = require('./backup')
-const Stats = require('./stats')
 
 module.exports = async function graphqlServer (ssb) {
   const PORT = (ssb.config.graphql && ssb.config.graphql.port) || env.graphql.port
@@ -45,7 +44,6 @@ module.exports = async function graphqlServer (ssb) {
 
   const invite = Invite(ssb)
   const backup = Backup(ssb)
-  const stats = Stats(ssb)
 
   let context = {}
   if (ssb.config.graphql && ssb.config.graphql.loadContext !== false) {
@@ -66,7 +64,6 @@ module.exports = async function graphqlServer (ssb) {
       author,
       groupInit,
       invite,
-      stats,
       backup
     ],
     context,
@@ -84,12 +81,12 @@ module.exports = async function graphqlServer (ssb) {
     cordova-bridge is used in Cordova, DO NOT REMOVE!
     The if statement is uncommented by patch-package in the mobile platform
   */
-  // if (process.env.PLATFORM === 'cordova') {
-  //   require('cordova-bridge').channel.post(
-  //     'initialized',
-  //     JSON.stringify({ started: true })
-  //   )
-  // }
+  if (process.env.PLATFORM === 'cordova') {
+    require('cordova-bridge').channel.post(
+      'initialized',
+      JSON.stringify({ started: true })
+    )
+  }
 
   return httpServer
 }
