diff --git a/node_modules/ssb-ahau/src/graphql-server.js b/node_modules/ssb-ahau/src/graphql-server.js
index 509ca8b..b14cc10 100644
--- a/node_modules/ssb-ahau/src/graphql-server.js
+++ b/node_modules/ssb-ahau/src/graphql-server.js
@@ -66,12 +66,12 @@ module.exports = function graphqlServer (sbot) {
       )
 
       /* cordova-bridge is not a known npm package? */
-      // if (process.env.PLATFORM === 'cordova') {
-      //   require('cordova-bridge').channel.post(
-      //     'initialized',
-      //     JSON.stringify({ started: true })
-      //   )
-      // }
+      if (process.env.PLATFORM === 'cordova') {
+        require('cordova-bridge').channel.post(
+          'initialized',
+          JSON.stringify({ started: true })
+        )
+      }
     })
 
     sbot.close.hook((close, args) => {
