diff --git a/node_modules/ssb-ahau/src/graphql/index.js b/node_modules/ssb-ahau/src/graphql/index.js
index a80200e..53ee3f2 100644
--- a/node_modules/ssb-ahau/src/graphql/index.js
+++ b/node_modules/ssb-ahau/src/graphql/index.js
@@ -19,7 +19,7 @@ const Settings = require('@ssb-graphql/settings')
 
 const Author = require('./author')
 const Backup = require('./backup')
-const Stats = require('./stats')
+// const Stats = require('./stats')
 
 module.exports = function graphqlServer (ssb, cb) {
   const PORT = (ssb.config.graphql && ssb.config.graphql.port) || env.graphql.port
@@ -40,7 +40,7 @@ module.exports = function graphqlServer (ssb, cb) {
 
   const invite = Invite(ssb)
   const backup = Backup(ssb)
-  const stats = Stats(ssb)
+  // const stats = Stats(ssb)
 
   if (ssb.config.graphql && ssb.config.graphql.loadContext === false) {
     return next(null)
@@ -61,7 +61,7 @@ module.exports = function graphqlServer (ssb, cb) {
       settings,
       author,
       invite,
-      stats,
+      // stats,
       backup
     ])
 
@@ -120,12 +120,12 @@ module.exports = function graphqlServer (ssb, cb) {
         cordova-bridge is used in Cordova, DO NOT REMOVE!
         The if statement is uncommented by patch-package in the mobile platform
       */
-      // if (process.env.PLATFORM === 'cordova') {
-      //   require('cordova-bridge').channel.post(
-      //     'initialized',
-      //     JSON.stringify({ started: true })
-      //   )
-      // }
+      if (process.env.PLATFORM === 'cordova') {
+        require('cordova-bridge').channel.post(
+          'initialized',
+          JSON.stringify({ started: true })
+        )
+      }
 
       cb(null, httpServer)
     })
