diff --git a/node_modules/urlpattern-polyfill/dist/urlpattern.cjs b/node_modules/urlpattern-polyfill/dist/urlpattern.cjs
index 1199eec..1516c78 100644
--- a/node_modules/urlpattern-polyfill/dist/urlpattern.cjs
+++ b/node_modules/urlpattern-polyfill/dist/urlpattern.cjs
@@ -44,8 +44,8 @@ var Part = class {
     return this.name !== "" && typeof this.name !== "number";
   }
 };
-var regexIdentifierStart = /[$_\p{ID_Start}]/u;
-var regexIdentifierPart = /[$_\u200C\u200D\p{ID_Continue}]/u;
+var regexIdentifierStart = /[$_a-zA-Z]/;
+var regexIdentifierPart = /[$_a-zA-Z0-9\-]/;
 var kFullWildcardRegex = ".*";
 function isASCII(str, extended) {
   return (extended ? /^[\x00-\xFF]*$/ : /^[\x00-\x7F]*$/).test(str);
