diff --git a/node_modules/nodejs-mobile-cordova/src/android/build.gradle b/node_modules/nodejs-mobile-cordova/src/android/build.gradle
index 4e5783d..a1e429e 100644
--- a/node_modules/nodejs-mobile-cordova/src/android/build.gradle
+++ b/node_modules/nodejs-mobile-cordova/src/android/build.gradle
@@ -220,14 +220,14 @@ cdvPluginPostBuildExtras += { ->
                 description = "Building a native toolchain to compile nodejs-mobile native modules for ${abi_name}."
                 executable = "${ndk_bundle_path}/build/tools/make-standalone-toolchain.sh"
                 args "--toolchain=${temp_toolchain_name}-${temp_cc_ver}", "--arch=${temp_arch}", "--install-dir=${standalone_toolchain}", "--stl=libc++", "--force", "--platform=android-22"
-                outputs.file "${standalone_toolchain}"
+                outputs.dir "${standalone_toolchain}"
             }
 
             task "BuildNpmModules${abi_name}" (type:Exec) {
                 dependsOn "MakeToolchain${abi_name}"
                 dependsOn "CopyNodeProjectAssets${abi_name}"
                 inputs.file "${rootProject.buildDir}/nodejs-native-assets-temp-build/nodejs-native-assets-${abi_name}/copy.timestamp"
-                outputs.file "${rootProject.buildDir}/nodejs-native-assets-temp-build/nodejs-native-assets-${abi_name}/nodejs-project/"
+                outputs.dir "${rootProject.buildDir}/nodejs-native-assets-temp-build/nodejs-native-assets-${abi_name}/nodejs-project/"
                 description = "Building native modules for ${abi_name}."
                 workingDir "${rootProject.buildDir}/nodejs-native-assets-temp-build/nodejs-native-assets-${abi_name}/nodejs-project/"
                 commandLine npmCommandName, '--verbose', 'rebuild', '--build-from-source'
@@ -273,7 +273,7 @@ cdvPluginPostBuildExtras += { ->
             task "GenerateNodeNativeAssetsLists${abi_name}" {
                 dependsOn "CopyBuiltNpmAssets${abi_name}"
                 description "Generates a list for runtime copying"
-                inputs.file "${rootProject.buildDir}/nodejs-native-assets/nodejs-native-assets-${abi_name}/"
+                inputs.dir "${rootProject.buildDir}/nodejs-native-assets/nodejs-native-assets-${abi_name}/"
                 outputs.file "${rootProject.buildDir}/nodejs-native-assets/nodejs-native-assets-${abi_name}/file.list"
                 outputs.file "${rootProject.buildDir}/nodejs-native-assets/nodejs-native-assets-${abi_name}/dir.list"
                 doLast{
